src/resources/swagger.yaml

```diff
@@ -70,7 +70,7 @@ paths:
                   type: object
     patch:
       tags:
-        - Schemas
+        - Dictionary
```

Comment:
can we create a tag for dictionary migraiton only will be easier for admins ?

Reply:
can you also create a tag called Admin and tag all admin APIs like Patch Operations and migration, icgc import etc

Reply:
because I just noticed tags is an array so we can add many here

Reply:
Agree, it makes sense to keep then organized together :+1: